tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -55,14 +55,13 @@ generate_service_file() {
 Description=${description}
 
 [Service]
+ExecStartPre=/bin/mv ${log_file} ${log_file}-backup 
```

Comment:
Can it error out? Shall we make sure `log_file` exists before we `mv`?

Reply:
Like

```
[ -d "$log_file" ] && /bin/mv "${log_file}" "${log_file}-backup" 
```

Reply:
Done ✅